package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "verbose-express",
+  "version": "0.0.1",
+  "description": "Express implementation of Verbose CMS",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
Would be nice to have one test framework across the node api and the FE dashboard